_data/providers.json

```
@@ -433,6 +433,11 @@
       "name": "WitherHosting",
       "url": "https://witherhosting.com/",
       "description": "Get Geyser as a plugin. Put the port provided by the host in your config.yml for the Java and bedrock sections, or create a port in the ports manager on the panel and use that for bedrock with the main one for Java, and you will be able to connect! Refer to the hosts [support article](https://support.witherhosting.com/en/article/how-to-install-and-use-geysermc-1xn5l6v/) for more"
+    },
+    {
+      "name": "FREAKHOSTING",
+      "url": "https://freakhosting.com",
+      "description": "Get Geyser as a plugin. Use the same port as your Java server for the Bedrock port in your config and connect with that port."
```

```suggestion
      "description_template": "default"
```

Your description is pretty much the same as the default: https://github.com/GeyserMC/GeyserWiki/blob/master/_data/providers.json#L3

---

Do we need to provide a custom description or we can use the default one as that is the most suitable one ?

---

A custom description only has to be provided if the default template isn't accurate – what you initially provided is fundamentally the same as the default, so we can use the default template